octopus_sensing/devices/skinos_streaming.py

```diff
@@ -0,0 +1,264 @@
+
+        Parameters
+        -----------
+        message: Message
+                 a message object
+        '''
+        self._trigger = \
+            "{0}-{1}-{2}".format(message.type,
+                                 message.experiment_id,
+                                 str(message.stimulus_id).zfill(2))
+
+    def _stream_loop(self):
+        '''
+        Reads incoming data
+        '''
+        initial_flag = False
+        framesize = 15
+
+        try:
+            while True:
+                time.sleep(0.1)
+
+                result = self._serial.read_until(b':', 16)
+
+                if self._break_loop:
+                    self._stop_skinos()
+                    break
+                
+                ch1 = result[0:3]
+                ch2 = result[4:7]
+                ch3 = result[8:11]
+                ch4 = result[12:15]
+
+                (timestamp) = datetime.datetime.now()
+
+                if initial_flag is False:
+                    ch1_str = '000000'
+                    ch2_str = '000000'
+                    ch3_str = '000000'
+                    ch4_str = '000000'
+
+                    initial_flag = True
+
+                else:
+                    ch1_str = repr(ch1)
+                    ch2_str = repr(ch2)
+                    ch3_str = repr(ch3)
+                    ch4_str = repr(ch4)
+
+                ch_int_list = []
+                
+                ch_int_list.append(int(ch1_str[2:5], 16))
+                ch_int_list.append(int(ch2_str[2:5], 16))
+                ch_int_list.append(int(ch3_str[2:5], 16))
+                ch_int_list.append(int(ch4_str[2:5], 16))
+
+                for i in range(len(ch_int_list)):
+                    if ch_int_list[i] < 2048:
+                        ch_int_list[i] = ch_int_list[i] / 500
+                    
+                    else:
+                        ch_int_list[i] = - ((4096 - ch_int_list[i]) / 500)
+
+                print(ch_int_list)
+
+                self._client.sendto(result,(HOST, PORT))
+
+                #print(timestamp, ch1, ch2, ch3, ch4, self._trigger)
+
+                if self._trigger is not None:
+                    print("Skinos trigger")
+                    row = [timestamp,
+                           ch_int_list[0], 
+                           ch_int_list[1],
+                           ch_int_list[2],
+                           ch_int_list[3],
+                           self._trigger]
+                    self._trigger = None
+                else:
+                    row = [timestamp,
+                           ch_int_list[0],
+                           ch_int_list[1],
+                           ch_int_list[2],
+                           ch_int_list[3],
+                           self._trigger]
+                self._stream_data.append(row)
+
+        except KeyboardInterrupt:
+            self._stop_skinos()
+
+    def _stop_skinos(self):
+
+        print("All done")
```

Comment:
Please change it to something like `Skinos: stopped`. Just like your other logs, so it will be clear among all the other stuff Octopus Sensing is logging.